serverless/src/submitSwodlrOrder/__tests__/retrieveCMRGranules.test.js

```diff
@@ -131,4 +132,39 @@ describe('retrieveCMRGranules', () => {
       ]
     })
   })
+
+  test('sends all params to CMR', async () => {
+    nock(/cmr/)
+      .matchHeader('Authorization', 'Bearer access-token')
+      .post('/search/granules.json', (body) => {
+        const params = parseQueryString(body)
+
+        // Verify params are sent
+        expect(params.echo_collection_id).toBe('C2799438271-POCLOUD')
+        expect(params.readable_granule_name).toEqual(['*_011_424_027*'])
+        expect(params.page_num).toBe('1')
+
+        return true
+      })
```

Comment:
I don't think we need a new test in this file, but this block needs to be added to the existing tests, because right now they aren't really testing anything (you can change the input parameters and they still pass).